paper-fab-speed-dial-action.html

```diff
@@ -43,11 +43,13 @@
 				padding: 5px 10px;
 				border-radius: 3px;
 				margin-right: 20px;
+				@apply --paper-fab-speed-dial-action-label;
```

Comment:
My apologies! I added them all to master without thinking about the pull request consequences. I'll make these changes soon.